example/provider-local/shoot-workerless.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: core.gardener.cloud/v1beta1
+kind: Shoot
+metadata:
+  name: local
```

```suggestion
  name: local-wl
```
That you can start both example shoots in parallel in your local setup

---

You could add the new host to the docs too: https://github.com/gardener/gardener/blob/master/docs/deployment/getting_started_locally.md#accessing-the-shoot-cluster

Btw. the list in docs is not updated with the new/changed hostnames anyway.

---

This was in our list, but missed it. Will do.